packages/manager/apps/telecom/src/components/telecom/telephony/timeCondition/condition/edit/telephony-time-condition-condition-edit.controller.js

```diff
@@ -187,23 +189,23 @@ angular.module('managerApp').controller('voipTimeConditionConditionCtrl', functi
 
   /* ----------  TimeFrom and TimeTo  ----------*/
 
-  self.onTimeFromChange = function onTimeFromChange() {
+  $scope.$watch('$ctrl.model.timeFrom', () => {
```

Comment:
There's an `onChange` callback available for `oui-timepicker` to prevent you from `$scope.watch`

Reply:
`OnChange` is called before the `ngModel` changes. So it always captures the previous value onChange. Hence using `$scope.$watch` here.

Reply:
You can pass modelValue and get the updated value as the function parameter 
`<oui-timepicker model="$ctrl.eventsModel" on-change="$ctrl.onChange(modelValue)">
</oui-timepicker>`
```suggestion
 self.onTimeFromChange = function onTimeFromChange(modelValue) {
 // modelValue will be the updated value
 }
```